src/gateway/upgrade.md

```diff
@@ -43,9 +43,23 @@ affect your current installation.
 Amazon Linux 1 and Debian 8 (Jessie) containers and packages are deprecated and are no longer produced for new versions of {{site.base_gateway}}.
 
 Blue-green deployments:
-* **{{site.ee_product_name}}**: Blue-green upgrades from versions 2.1.x.x-2.7.x.x to 3.0.0.0 are not currently supported.
-  * This is a known issue planned to be fixed in the next 2.8.x.x release. When that version is released, 2.x users should upgrade to that version before beginning a blue-green upgrade to 3.0.0.0.
-* **{{site.ce_product_name}}**: Blue-green deployments are not supported for major versions, therefore they are not supported with upgrades from 2.x.x to 3.0.x.
+* Traditional mode: Blue-green upgrades from versions of 2.8.1(2.8.1.x for Enterprise) and below to 3.0.0(3.0.0.0 for Enterprise) are not currently supported for traditional mode.
+  * This is a known issue planned to be fixed in the next 2.8 release. When that version is released, 2.x users should upgrade to that version before beginning a blue-green upgrade to 3.0.
+* Hybrid mode: migration can be done by following steps:
```

Comment:
Does it make sense to describe this process for EE customers in this brief form?  it might be better to place this description in an internal document that is used by the sustaining team to help customers who actually need to do a blue-green upgrade of EE before we've made the 2.8.x helper release.

Reply:
> it might be better to place this description in an internal document that is used by the sustaining team to help customers who actually need to do a blue-green upgrade of EE before we've made the 2.8.x helper release.

We don't have/maintain internal documents for the sustaining team. If you're aware of any location of these docs, you're welcome to use + copy any of this content to add to them 😄

Reply:
> Does it make sense to describe this process for EE customers in this brief form? it might be better to place this description in an internal document that is used by the sustaining team to help customers who actually need to do a blue-green upgrade of EE before we've made the 2.8.x helper release.

Yes those steps are better put into `upgrade.md`.